src/search-injector/background/interface.js

```diff
@@ -0,0 +1,24 @@
+import { SEARCH_INJECTION_KEY } from '../constants'
+
+export async function fetchSearchInjection() {
+    const { searchInjection } = await browser.storage.local.get(
+        SEARCH_INJECTION_KEY,
+    )
+    // For old browsers do typeof searchInjection === "undefined"
+    if (searchInjection === undefined) {
```

Comment:
You can supply a default value like `browser.storage.local.get({ [SEARCH_INJECTION_KEY]: true })` then you just need to return that value. Shouldn't be a need to call `storeSearchInjection` if it's not defined yet.

Reply:
@poltak I need to make the store the function more generalised so i can use it to store more variables so I will change it any ways